html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html

```diff
@@ -189,6 +192,8 @@
  * [[Delete]]
  */
 addTest(function() {
+  assert_throws(null, function() { delete C[0]; }, "Can't delete cross-origin indexed property");
```

Comment:
Shouldn't all these tests be asserting something about the type of exception thrown?  Presumably the spec specifies that...